frontend/uddhar-frontend/src/authentication/components/VolunteerDetails.jsx

```
@@ -0,0 +1,157 @@
+      handleChange({
+        target: {
+          name: "skills",
+          value: [...formData.skills, newSkill],
+        },
+      });
+
+      // Clear error when skills are added
+      if (errors.skills) {
+        setErrors((prev) => {
+          const newErrors = { ...prev };
+          delete newErrors.skills;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)